tsl/test/sql/cagg_joins.sql.in

```diff
@@ -153,6 +162,64 @@ SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
 FROM conditions JOIN devices USING (device_id)
 GROUP BY name, bucket;
 
+SELECT * FROM conditions_summary_daily_3_reorder ORDER BY bucket;
+
+CREATE MATERIALIZED VIEW conditions_summary_daily_3_reorder_using_realtime
+WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   MAX(temperature),
+   MIN(temperature),
+   name
+FROM conditions JOIN devices USING (device_id)
+GROUP BY name, bucket;
+
+SELECT * FROM conditions_summary_daily_3_reorder_using_realtime ORDER BY bucket;
+
+--Create CAgg with join and additional WHERE conditions
+CREATE MATERIALIZED VIEW conditions_summary_daily_more_conds
+WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   MAX(temperature),
+   MIN(temperature),
+   name
+FROM conditions JOIN devices ON conditions.device_id = devices.device_id
+WHERE conditions.city = devices.location AND
+      conditions.temperature > 28
```

Comment:
This should error out, no??? because you're adding another JOIN condition in the WHERE clause or am I missing something?